crates/concolor/src/lib.rs

```diff
@@ -30,6 +30,7 @@
 //! If you are providing a command line option for controlling color, just call
 //! ```rust
 //! let when = concolor::ColorChoice::Always;
+//! #[cfg(feature = "api_unstable")]
 //! concolor::set(when);
```

Comment:
The downside is this will show up in the user documentation.  While its useful to tell users of needed features (how `argfile` does this being the nicest), over-complicating examples with other details is a challenge.

Two possible options
- Prefix this with `# ` so rustdoc doesn't show it
- You can conditionalize the doc comment, see https://github.com/clap-rs/clap/blob/master/src/builder/arg.rs#L1613

(hadn't known or considered these options before which is why I leaned towards just requiring it)